tensorflow_datasets/core/features/feature.py

```diff
@@ -724,16 +724,25 @@ def get_inner_feature_repr(feature):
   else:
     return repr(feature)
 
+def _repr_html_batch(ex) -> str:
+  """Default HTML batch repr."""
+  batch_ex = ''
+  for value in ex:
+    batch_ex += _repr_html(value) + ', '
```

Comment:
Use `.join()` rather than `+=`. See: https://stackoverflow.com/questions/39312099/why-is-join-faster-than-in-python